bee-tangle/src/solid_entry_point.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! A SolidEntryPoint is a [`MessageId`](crate::MessageId) of a message even if we do not have them
```

Comment:
Missing word ?

Reply:
What word is missing?

Reply:
Sentence was weird to me, I'd better understand something like `A SolidEntryPoint is a [`MessageId`](crate::MessageId) of a message that is solid even if we do not have them...`